### DISCUSSION TITLE
Clean up stale branches?

### DISCUSSION THREAD

**Discussion by b1n-ch1kn** [FS-Online/Driverless-Competition-Simulator#329] [Ideas]
Not really an issue or anything, but if there are useful features hiding away in the many stale branches it might be cool to see what we can bring in.
Also clearing any out-of-date code would keep the repo tidy and less confusing.

**Comment by mateusz-lichota** [comment]
All of the stale branches were >400 commits behind, and none of them contained valuable functionality so I went ahead and deleted all of them
